### PR TITLE
Ccaldas/#1320 sync competition dev with master

### PR DESCRIPTION
**Pull request by carloscaldas**
Related to the issue: https://github.com/LBNL-UCB-STI/beam/issues/1320

Closed PR with comments https://github.com/LBNL-UCB-STI/beam/pull/1317

This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/1322)

### CODE REVIEW

**Comment by JustinPihony** [comment]
I'm not sure what happened in the other PR that was closed, but I'd at least like to address the potential issue with memory for using the log appender : https://github.com/LBNL-UCB-STI/beam/pull/1321#discussion_r259037106 I know it isn't on by default, but if it were accidentally kept on I'd wonder the impact on the overall.

**Comment by carloscaldas** [comment]
Hi @JustinPihony your comment is valid, but the task of this Appender was interrupted and is not worthy to spend time on it unless we want to use it. I will just add a comment related to your valid concern.

File: build.gradle [hunk 1]
```diff
@@ -108,6 +108,7 @@ dependencies {
     compile group: 'com.google.inject.extensions', name: 'guice-multibindings', version: '4.1.0'
     compile group: 'org.apache.commons', name: 'commons-collections4', version: '4.1'
     compile group: 'org.apache.commons', name: 'commons-math3', version: '3.5'
+    compile group: 'org.apache.commons', name: 'commons-text', version: '1.6'
```
Comment (wrashid) [inline build.gradle]: remove

**Comment by carloscaldas** [comment]
This PR is going to be closed in favor of smaller PRs that can be easily reviewed

**Pull request closed without merging**